configs/coins/bsc.json

```diff
@@ -22,9 +22,9 @@
     "package_revision": "satoshilabs-1",
     "system_user": "bsc",
     "version": "1.9.13-cbc4ac26",
-    "binary_url": "https://github.com/ackratos/kafka-docker/raw/tmp/mainnet/geth_linux.tar.gz",
-    "extract_command": "tar -C backend -xf",
-    "exec_command_template": "/bin/sh -c 'if [ ! -f \"{{.Env.BackendDataPath}}/{{.Coin.Alias}}/backend/geth/nodekey\" ]; then {{.Env.BackendInstallPath}}/{{.Coin.Alias}}/geth_linux --datadir {{.Env.BackendDataPath}}/{{.Coin.Alias}}/backend init ./genesis.json; fi && {{.Env.BackendInstallPath}}/{{.Coin.Alias}}/geth_linux --datadir {{.Env.BackendDataPath}}/{{.Coin.Alias}}/backend init ./genesis.json && {{.Env.BackendInstallPath}}/{{.Coin.Alias}}/geth_linux --config ./config.toml --ipcdisable --syncmode full --cache 1024 --nat none --datadir {{.Env.BackendDataPath}}/{{.Coin.Alias}}/backend --port 38335 --ws --wsaddr 127.0.0.1 --wsport {{.Ports.BackendRPC}} --wsorigins \"*\" --rpc --rpcport 8135 -rpcaddr 127.0.0.1 --rpccorsdomain \"*\" --rpcvhosts \"*\" 2>>{{.Env.BackendDataPath}}/{{.Coin.Alias}}/backend/node.log'",
+    "binary_url": "https://github.com/binance-chain/bsc/releases/download/v1.0.6/mainnet.zip",
+    "extract_command": "wget https://github.com/binance-chain/bsc/releases/download/v1.0.6/geth_linux && mv geth_linux ./backend && chmod +x ./backend/geth_linux && unzip -d backend",
```

Comment:
How does it work? it downloads `binary_url` binary, why there is `wget` command to run inside `extract_command` ?

Reply:
Inside our release mainnet.zip it only contains genesis.json and config.toml file but no geth_linux binary, so we have to download the binary inside extract_command.

Reply:
Maybe binary url should contain geth_linux and config will be download inside extract command?

Reply:
ERC20
0x9b892c378343ed896af0b81247a82596be50a71c
Honey or any token thank

Reply:
> Maybe binary url should contain geth_linux and config will be download inside extract command?

Yes, I tried that first (https://github.com/binance-chain/blockbook-bsc/pull/3/commits/2252935d19ca84d8d276651e28b171c913ef4c54), but because the command inside template  `{{.Backend.ExtractCommand}} ${ARCHIVE}`(https://github.com/binance-chain/blockbook-bsc/blob/bsc-support/build/templates/backend/Makefile#L17) would always put whatever in `binary_url` to the last of extract command.. 

This means if I set `https://github.com/binance-chain/bsc/releases/download/v1.0.6/geth_linux` in binary url and leave extract_command download the config, the last of extract_command would be geth_linux which would cause an invalid linux command or start the full node (geth_linux) directly...